sgx/trusted/include/kmyth_enclave.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _KMYTH_ENCLAVE_H_
```

Comment:
Should we rename this file something like ```kmyth_enclave_trusted_h``` to indicate in the filename that it's trusted functionality (and shouldn't be needed in untrusted code)?

Reply:
This is a good suggestion. I renamed the file to kmyth_enclave_trusted.h.